seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -40,6 +40,7 @@ angular.module('BE.seed.controller.menu', [])
       $scope.narrow_controller = false;
       $scope.wide_controller = false;
       $scope.username = window.BE.username;
+      $scope.logged_in = $scope.username.length > 0;
```

Comment:
By adding this we can render the sidebar and topbar without the user being logged in